web-ui/src/main/resources/catalog/style/gn_viewer.less

```diff
@@ -176,8 +178,25 @@
     top: auto;
     bottom: calc(~"50px + 1em");
     position: fixed;
+    .badge-2d, .badge-3d {
+      background-color: @brand-danger;
+      z-index: 9001;
+    }
+    .badge-left {
+      margin: -30px 0 0 -34px;
+    }
+    .badge-right {
+      margin-top: -30px;
```

Comment:
Is this line needed, since the element is hidden anyway?

Reply:
It's needed because there are 2 badges, on both sides of the icon. This one needs because it's only used on smaller screens.